### PR TITLE
Fix exe

### PR DESCRIPTION
**Pull request by ocefpaf**
Should help with https://github.com/conda-forge/staged-recipes/pull/3160#issuecomment-310537965

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -9,9 +9,6 @@ nmake /f makefile.vc
 nmake INSTDIR=%LIBRARY_PREFIX% /f makefile.vc install-all
 if errorlevel 1 exit 1
 
-move %LIBRARY_PREFIX%\bin\*.* %PREFIX%
```
Comment (ocefpaf) [inline recipe/bld.bat:12]: I am not sure why we had this line here but it is copying more than it should. Also, ` %LIBRARY_PREFIX%\bin` is in the path, so we don't really need it.

ping @gillins

Comment (jschueller) [reply recipe/bld.bat:12]: cs2cs.exe, geod.exe, nad2bin.exe and proj.exe are part of proj4 though

**Comment by gillins** [comment]
Yep that's fine - but they should stay in Library/bin... i think the install step in the line before should be doing the right thing.

**Comment by jschueller** [comment]
Yes, I see proj exe is tested so it's fine.

**Review by jschueller — APPROVED** [review]

**Pull request merged**